shell/assets/translations/en-us.yaml

```
@@ -4286,9 +4287,11 @@ plugins:
     title: Error loading extension
     message: Could not load extension code
     generic: Extension error
-    api: This Extension is not compatible with the current Extensions API version
-    host: This Extension is not compatible with this application
-    version: This Extension is not compatible with this version of Rancher
+    apiAnnotationMissing: Unable to load Extension. The version installed does not have the mandatory annotation 'catalog.cattle.io/ui-extensions-version' from Rancher 2.10 and onwards
```

```suggestion
    apiAnnotationMissing: Unable to load Extension. The version installed is not compatible with the current Extensions API Version (Extension is missing the annotation 'catalog.cattle.io/ui-extensions-version' which implies incompatibility with Rancher 2.10 and onwards)
```

This will steer extension developers away from just sticking the annotation in